README.md

```diff
@@ -48,13 +48,20 @@ ArcticDB is designed from the outset to be resilient; there is no single point o
 
 ### Storage compatibility
 
-
 |                       | Linux | Windows | Mac |
 | --------------------- | - | - | - |
 | S3                 | ✔️ | ✔️ | ✔️ |
-| LMDB               | ✔️ | Beta | ✔️ |
+| LMDB               | ✔️ | ✔️ | ✔️ |
 | Azure Blob Storage | ✔️ | ✔️ | ➖ |
 
+We have tested against the following S3 backends:
+
+- AWS S3
+- VAST Data S3
+- Pure Storage S3
+- Ceph
+- MinIO on Linux
```

Comment:
Should we note that MinIO is currently not working on Windows?

Reply:
I just left it like this so that the `README.md` landing page is crisp to read

Reply:
Scality S3 as well.
```suggestion
- MinIO on Linux
- Scality S3
```